Workspace at latest with vue app, unable to add any npm package

### Current Behavior

After setting up a workspace with a vite/vue app I cant install other npm packages.
Somehow the generator was able to install requirements and running serve and test works.

But any other npm install or just install all will fail.

I tried --legacy-peer-deps etc and also manually playing around with versions both just cascaded into more and more errors.

Note:
I also tries this via an online ide (replit) and I had the same issue come up

### Expected Behavior

Be able to install packages

### GitHub Repo

running the create-workspace with vite/vue would be just as fast

### Steps to Reproduce

1.
Setup a workspace with vue with vite

Test that server and tests work - for me they did

then just try installing another npm package or run `npm install`

### Nx Report

```shell
Node           : 20.10.0
OS             : darwin-arm64
Native Target  : aarch64-macos
npm            : 10.2.3

nx                 : 20.5.0
@nx/js             : 20.5.0
@nx/eslint         : 20.5.0
@nx/workspace      : 20.5.0
@nx/devkit         : 20.5.0
@nx/eslint-plugin  : 20.5.0
@nx/playwright     : 20.5.0
@nx/vite           : 20.5.0
@nx/vue            : 20.5.0
@nx/web            : 20.5.0
typescript         : 5.7.3
---------------------------------------
Registered Plugins:
@nx/eslint/plugin
@nx/vite/plugin
@nx/playwright/plugin
```

### Failure Logs

```shell
npm install                                                                           ✔  base  
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @vitejs/plugin-vue@4.6.2
npm ERR! Found: vite@6.2.1
npm ERR! node_modules/vite
npm ERR!   dev vite@"^6.0.0" from the root project
npm ERR!   peer vite@"^5.0.0 || ^6.0.0" from @nx/vite@20.5.0
npm ERR!   node_modules/@nx/vite
npm ERR!     dev @nx/vite@"20.5.0" from the root project
npm ERR!     @nx/vite@"20.5.0" from @nx/vue@20.5.0
npm ERR!     node_modules/@nx/vue
npm ERR!       dev @nx/vue@"20.5.0" from the root project
npm ERR!   3 more (@vitest/mocker, vite-node, vitest)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^4.0.0 || ^5.0.0" from @vitejs/plugin-vue@4.6.2
npm ERR! node_modules/@vitejs/plugin-vue
npm ERR!   dev @vitejs/plugin-vue@"^4.5.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: vite@5.4.14
npm ERR! node_modules/vite
npm ERR!   peer vite@"^4.0.0 || ^5.0.0" from @vitejs/plugin-vue@4.6.2
npm ERR!   node_modules/@vitejs/plugin-vue
npm ERR!     dev @vitejs/plugin-vue@"^4.5.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Package Manager Version

_No response_

### Operating System

- [x] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information

If this bug report is bad my apologies, it's just I've spend a fair amount of time trying to resolve getting nowhere.
Keen to start using nx but have hit this wall immediately.

--- a/docs/generated/manifests/nx-api.json
+++ b/docs/generated/manifests/nx-api.json
@@ -5839,6 +5839,16 @@
       }
     },
     "migrations": {
+      "/nx-api/vue/migrations/20.7.1-package-updates": {
+        "description": "",
+        "file": "generated/packages/vue/migrations/20.7.1-package-updates.json",
+        "hidden": false,
+        "name": "20.7.1-package-updates",
+        "version": "20.7.1-beta.0",
+        "originalFilePath": "/packages/vue",
+        "path": "/nx-api/vue/migrations/20.7.1-package-updates",
+        "type": "migration"
+      },
       "/nx-api/vue/migrations/19.4.3-package-updates": {
         "description": "",
         "file": "generated/packages/vue/migrations/19.4.3-package-updates.json",

--- a/docs/generated/packages-metadata.json
+++ b/docs/generated/packages-metadata.json
@@ -5806,6 +5806,16 @@
     "migrations": [
       {
         "description": "",
+        "file": "generated/packages/vue/migrations/20.7.1-package-updates.json",
+        "hidden": false,
+        "name": "20.7.1-package-updates",
+        "version": "20.7.1-beta.0",
+        "originalFilePath": "/packages/vue",
+        "path": "vue/migrations/20.7.1-package-updates",
+        "type": "migration"
+      },
+      {
+        "description": "",
         "file": "generated/packages/vue/migrations/19.4.3-package-updates.json",
         "hidden": false,
         "name": "19.4.3-package-updates",

--- a/docs/generated/packages/vue/migrations/20.7.1-package-updates.json
+++ b/docs/generated/packages/vue/migrations/20.7.1-package-updates.json
@@ -1,0 +1,20 @@
+{
+  "name": "20.7.1-package-updates",
+  "version": "20.7.1-beta.0",
+  "packages": {
+    "vue": { "version": "^3.5.13", "alwaysAddToPackageJson": false },
+    "vue-tsc": { "version": "^2.2.8", "alwaysAddToPackageJson": false },
+    "vue-router": { "version": "^4.5.0", "alwaysAddToPackageJson": false },
+    "@vitejs/plugin-vue": {
+      "version": "^5.2.3",
+      "alwaysAddToPackageJson": false
+    }
+  },
+  "aliases": [],
+  "description": "",
+  "hidden": false,
+  "implementation": "",
+  "path": "/packages/vue",
+  "schema": null,
+  "type": "migration"
+}

--- a/packages/vue/migrations.json
+++ b/packages/vue/migrations.json
@@ -1,6 +1,27 @@
 {
   "generators": {},
   "packageJsonUpdates": {
+    "20.7.1": {
+      "version": "20.7.1-beta.0",
+      "packages": {
+        "vue": {
+          "version": "^3.5.13",
+          "alwaysAddToPackageJson": false
+        },
+        "vue-tsc": {
+          "version": "^2.2.8",
+          "alwaysAddToPackageJson": false
+        },
+        "vue-router": {
+          "version": "^4.5.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@vitejs/plugin-vue": {
+          "version": "^5.2.3",
+          "alwaysAddToPackageJson": false
+        }
+      }
+    },
     "19.4.3": {
       "version": "19.4.3-beta.0",
       "packages": {

--- a/packages/vue/src/utils/versions.ts
+++ b/packages/vue/src/utils/versions.ts
@@ -1,13 +1,13 @@
 export const nxVersion = require('../../package.json').version;
 
 // vue core
-export const vueVersion = '^3.3.4';
-export const vueTscVersion = '^2.0.0';
-export const vueRouterVersion = '^4.2.4';
+export const vueVersion = '^3.5.13';
+export const vueTscVersion = '^2.2.8';
+export const vueRouterVersion = '^4.5.0';
 
 // test deps
-export const vueTestUtilsVersion = '^2.4.1';
-export const vitePluginVueVersion = '^4.5.0';
+export const vueTestUtilsVersion = '^2.4.6';
+export const vitePluginVueVersion = '^5.2.3';
 
 // linting deps
 export const vueEslintConfigPrettierVersion = '7.1.0';

